backend/src/test/java/gov/cdc/usds/simplereport/api/ApiUserManagementTest.java

```diff
@@ -55,7 +55,7 @@ void whoami_standardUser_okResponses() {
     assertUserCanAccessExactFacilities(who, Set.of(TestUserIdentities.TEST_FACILITY_2));
     assertLastAuditEntry(
         TestUserIdentities.STANDARD_USER,
-        "current-user-query-operation",
+        "WhoDat",
```

Comment:
non-standard naming convention for graphql query action.

Reply:
Oh, hadn't thought about that. Probably worth fixing it to not jump out as weird.